nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -368,6 +368,8 @@ $(CsWinRTInternalProjection)
       <!-- Get the target platform (we either use 'Platform', if set and not 'AnyCPU', or fallback to 'PlatformTarget' ) -->
       <_StubExePlatform Condition="'$(_StubExePlatform)' == '' AND '$(Platform)' != '' AND '$(Platform)' != 'AnyCPU'">$(Platform)</_StubExePlatform>
       <_StubExePlatform Condition="'$(_StubExePlatform)' == '' AND '$(PlatformTarget)' != ''">$(PlatformTarget)</_StubExePlatform>
+
+      <_StubExeWin32ManifestPath Condition="'$(Win32Manifest)' != ''">$([System.IO.Path]::Combine($(MSBuildProjectDirectory), $(Win32Manifest)))</_StubExeWin32ManifestPath>
```

Comment:
Do we know if Win32Manifest is just the file name or the full path?

Reply:
It's a file name relative to the project directory.

Reply:
(That's why I'm combining it like this)